tests/smoke-tests/test-randomness-consistency.ts

```diff
@@ -343,14 +343,26 @@ function isRandom(bytes: Uint8Array) {
 
 // Tests if byte output is independent
 function chiSquareTest(bytes: Uint8Array) {
-  let chiSquared = 0;
-  // expected value is expected average of [u8; 32]
-  const expectedValue = 128;
-  // degrees of freedom is 32 - 1 = 31, alpha is 0.05
-  // chi.pdf(31, 0.05) = 44.985
+  let chiSquared = 0.0;
+  let numOnes = 0;
+  // expected value of 256 coin flips:
+  const expectedValue = 256 / 2;
+  // degrees of freedom is 256 - 1 = 255, alpha is 0.05
+  // chi.pdf(31, 0.05) = 287.882 (TODO: use precise value; this is from 250 in following table)
   // https://en.wikibooks.org/wiki/Engineering_Tables/Chi-Squared_Distibution
-  const pValue = 44.985;
-  bytes.forEach((a) => (chiSquared += ((a - expectedValue) ^ 2) / expectedValue));
+
+  // count occurences of ones
+  const pValue = 286.882;
+  bytes.forEach((a) => {
+    numOnes += [...a.toString(2)].filter((x) => x === "1").length;
+  });
```

Comment:
this is iterating for each byte `u8` of a `[u8; 32]`, it is not going through each bit

The input `bytes` type is `Uint8Array`.